etrago/tools/plot.py

```diff
@@ -1153,6 +1306,119 @@ def nodal_production_balance(network, timesteps, scaling=0.00001):
 
     """
 
+    def import_gen_from_links(network):
```

Comment:
It looks like this function is defined multiple times in this file (and actually it was already defined in another module of eTraGo if I'm not mistaken). Is that intended?

Reply:
It was intended, but the function in pf_post_lopf is used instead in the pro of clarity. Commit 28204f1 and 5f1d03e.